app.ts

```diff
@@ -0,0 +1,187 @@
+"use strict";
+window.onload = () => { onloadValues() };
```

Comment:
Rather write it like this - burying this single method call does not read cleanly.
```ts
window.onload = onloadValues();
```
Also, my personal preference is that you call this _onLoadValues_ (note the casing).